File: 상한규/백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a, b, v, d;
    scanf("%d %d %d", &a, &b, &v);
    d = ceil((v-a)/(double)(a-b)) + 1;
    printf("%d\n", d);
}

File: 상한규/백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringTokenizer st = new StringTokenizer(br.readLine());
		
		int A = Integer.parseInt(st.nextToken());
		int B = Integer.parseInt(st.nextToken());
		int V = Integer.parseInt(st.nextToken());
		int count = 1;
		
		count += Math.ceil((double)(V - A) / (A - B));
		
		System.out.println(count);
	}
}